### PR TITLE
test: thinbackup plugin

### PR DESCRIPTION
**Pull request by yanksyoon**
Applicable spec: N/A

### Overview

Test the backup of thinBackup plugin.

### Rationale

Commonly used plugin support.

### Juju Events Changes

None.

### Module Changes

None.

### Library Changes

None.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: tests/integration/test_plugins.py [hunk 1]
```diff
@@ -253,3 +253,39 @@ async def test_blueocean_plugin(ops_test: OpsTest, unit_web_client: UnitWebClien
...
+        f"{unit_web_client.web}/manage/thinBackup/backupManual"
+    )
+    unit_web_client.client.requester._get_crumb_data()
+    res.raise_for_status()
+
+    ret, stdout, stderr = await ops_test.juju(
+        "ssh", "--container", "jenkins", unit_web_client.unit.name, "ls", backup_path
+    )
+    assert ret == 0, f"Failed to ls backup path, {stderr}"
+    # The backup folder is of format FULL-<backup-date>
```
Comment (arturo-seijas) [inline tests/integration/test_plugins.py:290]: Can you add this comment to the assertion instead?

Comment (yanksyoon) [reply tests/integration/test_plugins.py]: Done!

File: tests/integration/test_plugins.py [hunk 2]
```diff
@@ -287,5 +287,4 @@ async def test_thinbackup_plugin(ops_test: OpsTest, unit_web_client: UnitWebClie
         "ssh", "--container", "jenkins", unit_web_client.unit.name, "ls", backup_path
     )
     assert ret == 0, f"Failed to ls backup path, {stderr}"
-    # The backup folder is of format FULL-<backup-date>
-    assert "FULL" in stdout
+    assert "FULL" in stdout  # The backup folder is of format FULL-<backup-date>
```
Comment (arturo-seijas) [inline tests/integration/test_plugins.py:290]: Sorry, I  meant `assert "FULL" in stdout, The backup folder is not of format FULL-<backup-date>`

Comment (yanksyoon) [reply tests/integration/test_plugins.py:290]: Just to make sure, you mean like this?

```
# assert "FULL" in stdout, the backup folder is of format FULL-<backup-date>
assert "FULL" in stdout
```

Comment (arturo-seijas) [reply tests/integration/test_plugins.py]: No, I mean removing the comment and instead adding the explanation to the `assert` statement

Comment (yanksyoon) [reply tests/integration/test_plugins.py]: Done!

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by amandahla — APPROVED** [review]
LGTM

**Pull request merged**